packages/patternfly-4/react-core/src/components/DataToolbar/DataToolbar.tsx

```diff
@@ -67,6 +67,12 @@ export class DataToolbar extends React.Component<DataToolbarProps, DataToolbarSt
     if (this.isToggleManaged()) {
       window.addEventListener('resize', this.closeExpandableContent);
     }
+    if (!process.env.JEST_WORKER_ID) {
```

Comment:
how about mimicking this https://github.com/patternfly/patternfly-react/blob/cfd75c376268409cf8605a459c2ba94cc799c6ae/packages/patternfly-4/react-core/src/components/SimpleList/SimpleList.tsx#L47